tests/slow_tests/test_async.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+from threading import Thread
+
+from toloka.async_client import AsyncTolokaClient
+from urllib3 import Retry
+
+
+def test_async_client_survives_event_loop_change(connection_error_server_url, retries_before_response, fake_requester):
+    toloka_client = AsyncTolokaClient(
+        'fake-token',
+        url=connection_error_server_url,
+        retries=Retry(retries_before_response, status_forcelist={500}, backoff_factor=0),
+        retry_quotas=None,
+        timeout=0.5,
+    )
+
+    loop = asyncio.get_event_loop()
+    assert loop.run_until_complete(toloka_client.get_requester()) == fake_requester
+    loop.close()
+    loop = asyncio.new_event_loop()
+    assert loop.run_until_complete(toloka_client.get_requester()) == fake_requester
+
+
+def get_requester(result: list, client: AsyncTolokaClient):
+    result.append(asyncio.run(client.get_requester()))
+
+
+def test_async_client_can_be_multithreaded(connection_error_server_url, retries_before_response, fake_requester):
+    toloka_client = AsyncTolokaClient(
+        'fake-token',
+        url=connection_error_server_url,
+        retries=Retry(retries_before_response, status_forcelist={500}, backoff_factor=0),
```

Comment:
Please use `retryer_factory` parameter instead: [docstring about it](https://github.com/Toloka/toloka-kit/blob/main/src/client/__init__.py#L193)